preprocessors/togeojson-gpx.preprocessor.js

```diff
@@ -102,6 +102,14 @@ module.exports = function(infile, outdirectory, callback) {
       });
     });
 
+    var archivedOriginal = path.join(outdirectory, '/archived.gpx');
+    digest(infile, function(err, callback) {
```

Comment:
I had an inkling that including the `digest` step was redundant, thanks for confirming. I have removed this step.